test/fakeintake/client/client.go

```diff
@@ -841,6 +841,25 @@ func (c *Client) GetOrchestratorResources(filter *PayloadFilter) ([]*aggregator.
 	return orchs, nil
 }
 
+// GetLastOrchestratorResourcesPayloadAPIKeys fetches fakeintake on `/api/v2/orch` endpoint and returns
+// the API key of the last received orchestrator payload
```

Comment:
The comment doesn't match the implementation - implementation return all keys found in any paylod, not just the last one.,